src/tools/kata-ctl/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+[package]
+name = "kata-ctl"
+version = "0.0.0"
+authors = ["The Kata Containers community <kata-dev@lists.katacontainers.io>"]
+edition = "2021"
```

Comment:
Hi @jodh-intel ! Amazing initiative btw!

Regarding the rust edition, it seems all the other rust components are still on 2018 edition. Could the kata-ctl be built also on that edition?